src/jit/compiler.h

```diff
@@ -8817,12 +8817,16 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
     static HelperCallProperties s_helperCallProperties;
 
-#if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
```

Comment:
I believe that it is the convention that "FEATURE" tags should always be used as "#if FEATURE_UNIX_AMD64_STRUCT_PASSING. That is, they can be defined as 0 meaning off - that wouldn't work in this case. I realize that you have not changed the semantics, but I'm not sure why you changed it if you are not fixing it.